unified-runtime/scripts/core/exp-command-buffer.yml

```diff
@@ -310,7 +310,7 @@ returns:
     - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
 --- #--------------------------------------------------------------------------
 type: function
-desc: "Decrement the command-buffer object's reference count and delete the command-buffer object if the reference count becomes zero."
+desc: "Decrement the command-buffer object's reference count and delete the command-buffer object if the reference count becomes zero. It will try synchronizing the command-buffer, hence it is legal to call it while command-buffer is still executing."
```

Comment:
"It will try synchronizing the command-buffer," makes it sound like this will happen even if the reference count isn't zero. So would flip round the two phrases, something like

> It is legal to call the entry-point while `hCommandBuffer` is still executing, which will block on completion if the reference count of `hCommandBuffer` becomes zero.

Reply:
That sounds better, thanks :+1: